docs/reference/aggregations/pipeline.asciidoc

```diff
@@ -133,6 +144,7 @@ used here to filter out buckets which contain no buckets for an inner terms aggr
 
 [source,js]
 --------------------------------------------------
+POST /sales/_search
```

Comment:
I *thought* we only supported POST for _search operations because of clients that do not support GET with a body, so in the docs maybe we should prefer GET?